src/components/Home.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { CqlLibraryRoutes } from "./cqlLibraryRoutes/CqlLibraryRoutes";
+import { ApiContextProvider } from "../api/ServiceContext";
+import useGetServiceConfig from "./config/useGetServiceConfig";
+
+export default function Home() {
+  const errorPage = <div>Error loading service config</div>;
+  const { config, error } = useGetServiceConfig();
+  const loadingState = <div>Loading...</div>;
+
+  const loadedState = (
+    <ApiContextProvider value={config}>
+      <CqlLibraryRoutes />
+    </ApiContextProvider>
```

Comment:
We need a timeout Handler.

Reply:
I can use timeout handler here but I think that It would be kind of redundant code, my suggestion is that rather than giving timeout handler to each and every project can't we just give it in the layout component so that it will be automatically added to all the projects.

Reply:
I have spoke to Matthew about it, he said there is okta token issue when implementing on the layout. So for now I have created a new folder in the madie-cql-library for timeouthandler and imported it into CqlLibraryRoutes component and used it